site/en/guide/create_op.md

```diff
@@ -47,16 +47,16 @@ To incorporate your custom op you'll need to:
     test the op in C++. If you define gradients, you can verify them with the
     Python `tf.test.compute_gradient_error`.
     See
-    [`relu_op_test.py`](https://www.tensorflow.org/code/tensorflow/python/kernel_tests/relu_op_test.py) as
+    [`relu_op_test.py`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/nn_ops/relu_op_test.py) as
     an example that tests the forward functions of Relu-like operators and
     their gradients.
 
 ### Prerequisites
 
 *   Some familiarity with C++.
 *   Must have installed the
-    [TensorFlow binary](../../install), or must have
-    [downloaded TensorFlow source](../../install/source.md),
+    [TensorFlow binary](https://github.com/tensorflow/docs/blob/master/site/en/install), or must have
```

Comment:
The original was a relative link for

https://www.tensorflow.org/install

Reverting:

```suggestion
    [TensorFlow binary](../../install), or must have
```